### PR TITLE
Ft/wisdom/advertise

### PR DESCRIPTION
**Pull request by Ajinormotor**
## Description

Please include a summary of the changes and the related issue.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Blossom1-chike** [comment]
@Ajinormotor please can you fix the conflicts on this branch also, and move your files in partials to the chunks folder and follow the naming convention

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+leading-[22px] text-[14px] font-[700] font-epilogue  "
+              >
+                Make a Payment
+              </button>
+            </div>
+          </div>
+        </div>
+
+        {/* Premium Plan */}
+        <div className="bg-div-custom w-[87%] border-[2px]  rounded-[20px]">
```
Comment (Blossom1-chike) [inline src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:93]: confirm that this outer border exists on figma

File: src/components/cards/advertisebenefit-card/AdvertiseBenefitCard.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const AdvertiseBenefitCard = (props: AdvertiseBenefitCardProps) => {
+  return (
+    <section className="flex flex-col items-center justify-center">
+      <div
+        className="md:w-[274px] md:h-[300px] h-[224px] w-[298px] bg-transparent 
+       md:bg-[#F9FDFF] md:p-[1rem] p-0 rounded-[8px] flex flex-col 
+       items-start justify-center"
+      >
+        <div className="w-[40px] h-[40px] bg-[#ECF2FE] flex items-center justify-center">
```
Comment (Blossom1-chike) [inline src/components/cards/advertisebenefit-card/AdvertiseBenefitCard.tsx:13]: there should be a border radius for this image, check with the design

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+            {/* second divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+
+            <div className="flex flex-col items-center justify-center  p-[10px]">
+              <h1 className="font-[700] text-[#002360] text-[30px] leading-[48px] font-epilogue">
+                #50, 000
+              </h1>
+              <button
+                className="bg-[#0046BF] w-[147px] h-[42px] gap-[10px] text-[#FFFFFF]
```
Comment (Blossom1-chike) [inline src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:156]: there is a button component that exisits, please replace this

File: public/affordable.svg
Comment (Blossom1-chike) [inline public/affordable.svg:1]: please put these svgs into the images folder and then import and export them in the index.ts file in that same folder for uniformity

File: src/chunks/advertise-template-chunk/advertise-benefit-template-chunk/AdvertiseBenefitChunk.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+
+      {/* mapping through with data */}
+      <div
+        className="grid grid-cols-1  md:grid-cols-2 lg:grid-cols-4 
+        md:gap-[30px] max-w-[1292px] w-[93%] md:items-center justify-center space-y-[0.6rem]"
+      >
+        {benefitdata.map((a: AdvertiseBenefitCardProps, i: number) => (
+          <AdvertiseBenefitCard
+            key={i}
```
Comment (Blossom1-chike) [inline src/chunks/advertise-template-chunk/advertise-benefit-template-chunk/AdvertiseBenefitChunk.tsx:29]: if this is a props key, add it to the type definition of this props

**Review by Blossom1-chike — CHANGES_REQUESTED** [review]
also, link the advertise page to the text on the footer

**Review by Ajinormotor — COMMENTED** [review]
i have made all changes required,
linking the page in footer, renaming the folder, input the correct color code, added the correct border-radius, moved the svg into images folder e.t c

**Review by Ajinormotor — COMMENTED** [review]
done with the  recommended changes still open for further corrections

**Comment by Blossom1-chike** [comment]
image
this is how this card looks on mobile, please look into this

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            ))}
+
+            {/* second divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+
+            <div className="flex flex-col items-center justify-center  gap-[10px] my-[2rem]  ">
+              <h1 className="font-[700] text-[#002360] text-[30px] leading-[48px] font-epilogue">
+                #20,000
+              </h1>
+              <button
```
Comment (Blossom1-chike) [inline src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:60]: there is a button component, please make use of this here

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+            ))}
+
+            {/* second divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+
+            <div className="flex flex-col items-center justify-center  gap-[10px] my-[2rem]">
+              <h1 className="font-[700] text-[#002360] md:text-[30px] md:leading-[48px] font-epilogue">
+                #50,000
+              </h1>
+              <button
```
Comment (Blossom1-chike) [inline src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:109]: there is a button component, please make use of this here

**Review by Blossom1-chike — CHANGES_REQUESTED** [review]
please check these upates and a comment on the mobile view

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx (hunk 5 above)
Comment (Ajinormotor) [reply src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:60]: i added the button to the make payment, are you saying i should replace the h1 with button?

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx (hunk 6 above)
Comment (Ajinormotor) [reply src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:109]: i added the button to the make payment, are you saying i should replace the h1 with button?

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx (hunk 5 above)
Comment (Ajinormotor) [reply src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:60]: i added the button to the make payment, are you saying i should replace the h1 with button also am lost?

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx (hunk 6 above)
Comment (Ajinormotor) [reply src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:109]: i added the button to the make payment, are you saying i should replace the h1 with button also am lost?

File: src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx (hunk 5 above)
Comment (Blossom1-chike) [reply src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx:60]: there is a button component in the components folder. i'm saying you should use that component

**Review by Ajinormotor — COMMENTED** [review]
Done with the necessary changes , The only thing left is the mobile background of benefit which needs clarification from the designer

**Review by Blossom1-chike — APPROVED** [review]

**Pull request merged**